starknet_py/tests/e2e/contract_interaction/declare_test.py

```diff
@@ -28,3 +28,22 @@ async def test_throws_when_cairo1_without_compiled_contract_casm_and_class_hash(
             compiled_contract=compiled_contract,
             resource_bounds=MAX_RESOURCE_BOUNDS,
         )
+
+
+@pytest.mark.asyncio
+async def test_declare(
```

Comment:
Shouldn't that be `test_declare_v3_with_tip`?:
```suggestion
async def test_declare(
```

Reply:
Added `v3` to the name, but I couldn't find a general `test_declare_v3` test like that (except for one used for docs) so I though maybe I can treat it as a "default" test.